src/lib_json/json_value.cpp

```diff
@@ -497,10 +497,18 @@ Value::~Value() {
   value_.uint_ = 0;
 }
 
-Value& Value::operator=(Value other) {
+Value& Value::operator=(const Value& other) {
+  Value tmp(other);
+  swap(tmp);
```

Comment:
Swapping into a temporary is wasted work.
I think we should rather write "real" operators rather than relying on swap.

I'll work on that now and send a different PR proposal.